test/util/client.go

```diff
@@ -36,15 +37,20 @@ func KubeConfigPath() string {
 	return filepath.Join(GetBaseDir(), "openshift.local.config", "master", "admin.kubeconfig")
 }
 
-func GetClusterAdminKubeClient(adminKubeConfigFile string) (*kclientset.Clientset, error) {
+func GetClusterAdminKubeClient(adminKubeConfigFile string) (kclientset.Interface, error) {
 	c, _, err := configapi.GetKubeClient(adminKubeConfigFile, nil)
 	if err != nil {
 		return nil, err
 	}
 	return c, nil
 }
 
-func GetClusterAdminClient(adminKubeConfigFile string) (*client.Client, error) {
+func GetClusterAdminClient(adminKubeConfigFile string) (client.Interface, error) {
+	return GetClusterAdminClientRaw(adminKubeConfigFile)
+}
+
+// do not use this func unless you need to do raw REST or other low level operations
+func GetClusterAdminClientRaw(adminKubeConfigFile string) (*client.Client, error) {
```

Comment:
@liggitt This allowed me to get around the other type casts in the places where we needed the raw client.